### DISCUSSION TITLE
Welcome to Upendo-Dnn-PageManager Discussions!

### DISCUSSION THREAD

**Discussion by WillStrohl** [UpendoVentures/Upendo-Dnn-PageManager#61] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HZass** [comment]
Are there any pre-requisite modules/libraries that need to be installed to make this module work correctly?  My website is running Porto templates, Azure SQL Server and DNN 9.4.4.

I see no errors attributable to this module; but I get the headings to flash by and then no file names or other information.

**Reply by WillStrohl** [reply]
That might be great to ask as a "New" discussion.  There shouldn't be any issue with the information you presented so far.  You may want to try creating a test page that uses a core DNN theme and no other modules to see if you have any other issues.
